frontend/src/tests/pages/AdminUsersPage.test.js

```diff
@@ -5,20 +5,28 @@ import AdminUsersPage from "main/pages/AdminUsersPage";
 import usersFixtures from "fixtures/usersFixtures";
 import { apiCurrentUserFixtures } from "fixtures/currentUserFixtures";
 import { systemInfoFixtures } from "fixtures/systemInfoFixtures";
+import mockConsole from "jest-mock-console";
 
 import axios from "axios";
 import AxiosMockAdapter from "axios-mock-adapter";
 
-describe("AdminUsersPage tests",  () => {
-    const queryClient = new QueryClient();
-    test("renders without crashing on two users", async () => {
+describe("AdminUsersPage tests", () => {
 
-        var axiosMock = new AxiosMockAdapter(axios);
+    var axiosMock = new AxiosMockAdapter(axios);
+
+    const resetAxiosMock = () => {
+        axiosMock.reset();
+        axiosMock.resetHistory();
         axiosMock.onGet("/api/currentUser").reply(200, apiCurrentUserFixtures.userOnly);
         axiosMock.onGet("/api/systemInfo").reply(200, systemInfoFixtures.showingNeither);
-        axiosMock.onGet("/api/admin/users").reply(200, usersFixtures.threeUsers);
+    }
```

Comment:
Can we replace this with `beforeEach` ([docs](https://jestjs.io/docs/setup-teardown)) so we don't have to explicitly call it in tests?